packages/evm/src/stack.ts

```diff
@@ -83,16 +83,17 @@ export class Stack {
    * @param position - Index of item from top of the stack (0-indexed)
    */
   swap(position: number) {
-    if (this._store.length <= position) {
+    if (this._len <= position) {
       throw new EvmError(ERROR.STACK_UNDERFLOW)
     }
 
-    const head = this._store.length - 1
-    const i = this._store.length - position - 1
+    const head = this._len - 1
+    const i = head - position
+    const sto = this._store
```

Comment:
Is `sto` just short for `store`?  It wasn't obvious to me at first.

Reply:
Yes, and I am not sure if "caching" things actually improves something, but in my head it should somewhat improve performance (instead of doing key -> value lookup multiple times on an object where you know that obj[key] has not yet changed)

Reply:
Have cleared this up